addons/sourcemod/scripting/ProxyKiller.sp

```diff
@@ -5,6 +5,10 @@
 #include <SteamWorks>
 #include <ProxyKiller>
 
+#undef REQUIRE_PLUGIN
+#tryinclude <vip_core>
```

Comment:
remove the vip stuff from here and instead do the check here:
- https://github.com/srcdslab/sm-plugin-PlayerManager/blob/58f6f5a043714a5869964e3c9e145190e1b9501e/addons/sourcemod/scripting/PlayerManager.sp#L73

Reply:
ProxyKiller_IgnoreFlags check is before client get the flags from VIP plugin.

Reply:
he does it inside `OnClientPostAdminCheck` so at that point all VIP players have their flags (maybe it doesnt work for free vip players because flags are not given at the right time):
- https://github.com/srcdslab/sm-plugin-ProxyKiller/blob/983df8172b00433a07969626c7498b69d4721176/addons/sourcemod/scripting/ProxyKiller.sp#L111
- https://sourcemod.dev/#/clients/function.OnClientPostAdminCheck

> Called once a client is authorized and fully in-game, and
after all post-connection authorizations have been performed.

Reply:
In case of map start as exemple, the vip database can not always connected before the first client. 
This client will be checked instead of ignored because he didn't receive the flags at this moment.

Reply:
so its just a problem with vip_core/vip module then because no matter which plugin we use, all players must have their flags in that part otherwise a lot of others functionalities will not work like torchlight, skins etc :frowning_face:

Reply:
rework this PR with this logic in mind and i will take some time in early 2024 to fix this vip sm flag issue (feel free to open the issue with a good description and link it here)

Reply:
> so its just a problem with a vip module by itself then because no matter which plugin we use, all players must have their flags in that part otherwise a lot of others functionalities will also not work like torchlight, skins etc ☹️

I didnt saw a good way to solve this issue. Any ideas?

Reply:
i think its linked to this issue:
- https://github.com/srcdslab/sm-plugin-VIP_FreeVIPGiveaway/issues/4

Reply:
Another solution will be to create a file directly in ProxyKiller, for 3rdpart plugin.